bindings/java/src/operator.rs

```diff
@@ -0,0 +1,241 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::collections::HashMap;
+use std::str::FromStr;
+
+use jni::objects::{JByteArray, JClass, JObject, JString};
+use jni::sys::{jboolean, jlong, jsize};
+use jni::JNIEnv;
+
+use opendal::{BlockingOperator, Operator, Scheme};
+
+use crate::macros::*;
+use crate::util;
+
+#[no_mangle]
+pub unsafe extern "system" fn Java_org_apache_opendal_Operator_getOperator(
+    mut env: JNIEnv,
+    _class: JClass,
+    input: JString,
+    params: JObject,
```

Comment:
We don't need this. It's not the correct direction. We should not build  operator from a map in language like c or java.

Reply:
I only move this from lib.rs to operator.rs. I recommend refactoring in the next pr, this is not in the scope of the current pr.